Make startup command configurable (#505)
trinodb/trino-gateway: docs/installation.md

is recommended to protect the  database credentials required for this 
configuration.

> By default, the Trino Gateway process is started with the command 

Reviewer: ```suggestion
By default, the Trino Gateway process is started with the following command:

```

```shell
java -XX:MinRAMPercentage=80.0 -XX:MaxRAMPercentage=80.0 -jar /usr/lib/trino/gateway-ha-jar-with-dependencies.jar /etc/gateway/config.yaml
```

> This can be customized through the `command` node. This node accepts a list, which 

Reviewer: ```suggestion

You can customize details with the `command` node. It accepts a list, that 
```

> should begin with an executable such as `java` or `bash`. The following list 

Reviewer: ```suggestion
must begin with an executable such as `java` or `bash` that is available on the PATH. The following list 
```

elements are provided as arguments to the executable. It is not typically necessary

> to modify this node.

Reviewer: ```suggestion
to modify this node. You can use it to change of JVM startup parameters to control memory settings and other aspects, or to use other configuration file names.
```
